### PR TITLE
feat(cli): add test lib to forge init

### PR DESCRIPTION
**Pull request by korboismoe**
Similar to what [`dapp init`](https://github.com/dapphub/dapptools/blob/master/src/dapp/libexec/dapp/dapp-init) does.

### CODE REVIEW

File: assets/ContractTemplate.t.sol [hunk 1]
```diff
@@ -1,6 +1,12 @@
 // SPDX-License-Identifier: Unlicense
 pragma solidity 0.8.10;
 
-contract ContractTest {
+import "../../lib/ds-test/src/test.sol";
```
Comment (transmissions11) [inline assets/ContractTemplate.t.sol:4]: idt u want a relative import here?

**Review by gakonst — APPROVED** [review]
<s>LGTM. Thank you! Did you test that this works properly as a cli user?</s>

**Review by gakonst — APPROVED** [review]
Should work now. Before, the `lib/ds-test` was installed in `./lib/ds-test`, which would not work if you do `forge init my-project` because in that case you'd want it under `./my-project/lib/ds-test`.

**Comment by korboismoe** [comment]
Ah woops, forgot to test that. Thanks for the fix.

**Pull request merged**